### PR TITLE
Update copyright for existing files

### PR DESCRIPTION
**Pull request by satyakonduri**
# PR Submission checklist

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/491|

# Common PR Checklist:

- [x] Have you made sure that the code compiles?
- [ ] Have you commented your code, particularly in hard-to-understand areas
- [ ] Did you run tests in a real Kubernetes cluster?
- [ ] Have you maintained backward compatibility

## Description of your changes:

- Update incorrect copyright years.

### CODE REVIEW

File: fs_test.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright 2020 Dell EMC Corporation
+ * Copyright 2020-2021 Dell EMC Corporation
```
Comment (adarsh-dell) [inline fs_test.go:3]: Should it be 2022?

File: nfs.go [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright 2020 Dell EMC Corporation
+ * Copyright 2020-2021 Dell EMC Corporation
```
Comment (adarsh-dell) [inline nfs.go:3]: Do we need to add © symbol?

**Comment by adarsh-dell** [comment]
@satyakonduri In some of the files I can see that © symbol is missing, can we add it?

File: fs_types.go [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright 2020 Dell EMC Corporation
+ * Copyright © 2020-2022 Dell EMC Corporation
```
Comment (adarsh-dell) [inline fs_types.go:3]: Should it be Dell Inc. or its subsidiaries. All Rights Reserved instead of Dell EMC Corporation?

File: fs_test.go (hunk 1 above)
Comment (satyakonduri) [reply fs_test.go]: Done

File: nfs.go (hunk 2 above)
Comment (satyakonduri) [reply nfs.go]: done

File: fs_types.go (hunk 3 above)
Comment (satyakonduri) [reply fs_types.go]: done

**Review by suryagupta4 — APPROVED** [review]

**Review by AkshaySainiDell — APPROVED** [review]

**Pull request merged**